#77 insert method realisation

kortov opened the pull request:
#77

AVM82 reviewed (approved):
Looks good to me
As I see you do not use anymore String.format :)

Shtramak reviewed (approved):
Well done!

ivshapovalov commented on src/main/java/juja/sqlcmd/DataSet.java:
```diff
@@ -19,6 +19,10 @@ public void insertValue(int columnIndex, String string) {
     }
 
+    public String[] values() {
+        return row.clone();
```
Why you use clone() instead of arrayCopy() or smth?

kortov replied:
ArrayCopy() needs 5 parameters. 5, Carl!)

ivshapovalov commented on src/main/java/juja/sqlcmd/DatabaseManager.java:
```diff
@@ -71,6 +71,28 @@ public void close() throws SQLException {
 
+    boolean insert(String tableName, DataSet dataset) {
+        String dataSetFormatted = dataSetFormatted(dataset);
+        String sqlQuery = "INSERT INTO " + tableName + " VALUES(" + dataSetFormatted + ")";
```
Faster than 'format' but not so readable, isn't it?

kortov replied:
May be( Will be fixed

Shtramak replied:
In that case, when parameters goes sequentially I don't agree that 'format' will be much readable...

ivshapovalov commented on src/main/java/juja/sqlcmd/DatabaseManager.java:
```diff
@@ -71,6 +71,28 @@ public void close() throws SQLException {
+        }
+    }
+
+
```
Extra line

kortov replied:
yep

ivshapovalov commented on src/main/java/juja/sqlcmd/DatabaseManager.java:
```diff
@@ -71,6 +71,28 @@ public void close() throws SQLException {
+    private String dataSetFormatted(DataSet dataSet) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (String value : dataSet.values()) {
+            stringBuilder.append("'")
```
Maybe use local variable such as wrapperSymbol or smth?

kortov replied:
Yeah, I Thought about this. But it will overload the code, I think. 
Ok, I'll try to use it.

ivshapovalov commented on src/test/java/juja/sqlcmd/DatabaseManagerTest.java:
```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
     }
 
+    @Test
+    public void insertIntoTableWithExistingTable() throws SQLException {
```
Maybe insertWhenTableExists?

kortov replied:
I use "with" instead of "when", but it's a good point

ivshapovalov commented on src/test/java/juja/sqlcmd/DatabaseManagerTest.java:
```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
+    }
+
+    @Test
+    public void insertIntoTableWithNotExistingTable() {
```
Similar

ivshapovalov commented on src/test/java/juja/sqlcmd/DatabaseManagerTest.java:
```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
+    }
+
+    @Test
+    public void insertIntoTableWithExceedingNumberOfParameters() throws SQLException {
```
Maybe insertWhenExtraParameters?

kortov replied:
May be. I'll use your name, but for me "extra" is something good, like a bonus))

kortov commented on src/test/java/juja/sqlcmd/DatabaseManagerTest.java:
```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
+    }
+
+    @Test
+    public void insertWithNotExistingTable() {
```
test

kortov committed:
#77 readded insert method
- insert method
- formatter for DataSet values
- fixes acc comments

kortov committed:
#77 readded tests for insert method
- tests for insert method
- method for creating test table
- fixes acc comments

kortov committed:
#77 readded method for getting values from DataSet
